### PR TITLE
Moved webpack common into modules folder

### PR DESCRIPTION
**Pull request by Kirito-Excalibur**
Solves issue #384 

Moved the required folders and also refactored all imports.

### CODE REVIEW

File: src/renderer/coremods/badges/badge.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { filters, getByProps, waitForModule } from "../../modules/webpack";
-import { React } from "@common";
```
Comment (asportnoy) [inline src/renderer/coremods/badges/badge.tsx:2]: Why did you get rid off these? We would prefer `@common` over the full path

File: src/renderer/modules/common/api.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { filters, getExportsForProps, waitForModule } from "..";
+import { filters, getExportsForProps, waitForModule } from "./webpack";
```
Comment (asportnoy) [inline src/renderer/modules/common/api.ts:1]: Are you sure this is correct? I think it should be `../webpack`

**Review by asportnoy — CHANGES_REQUESTED** [review]

File: src/renderer/modules/common/api.ts (hunk 2 above)
Comment (Kirito-Excalibur) [reply src/renderer/modules/common/api.ts:1]: I found this to be correct.

**Comment by asportnoy** [comment]
@Kirito-Excalibur did you accidentally close this?

**Comment by Kirito-Excalibur** [comment]
Yeah😅😅😅

**Pull request closed without merging**